modules/ROOT/pages/release-notes.adoc

```diff
@@ -21,10 +22,10 @@ Our dedicated team will be here to guide you through this new updated mode, answ
 Service continuity is a key element of operational performance, so we are introducing a new native capability to help to ensure access to applications. Even when a node experiences downtime, users will remain seamlessly connected and will be able continue working without interruptions, ensuring smooth operations and increased user satisfaction.
 
 === Test toolkit V2
-Ensuring effective testing by keeping track of the scope performed and identifying errors is a real concern for our developers. To support this, a test report is generated, which is adding a visual representation of the test sequence directly on the BPMN diagrams. This provides a clearer and more intuitive way of tracking progress, collaborating effectively and aligning with business/process owners. A concrete step towards shorter iterations and streamlined project tracking.
+Ensuring https://documentation.bonitasoft.com/test-toolkit/latest/process-testing-overview.html[effective testing] by keeping track of the scope performed and identifying errors is a real concern for our developers. To support this, a test report is generated, which is adding a visual representation of the test sequence directly on the BPMN diagrams. This provides a clearer and more intuitive way of tracking progress, collaborating effectively and aligning with business/process owners. A concrete step towards shorter iterations and streamlined project tracking.
```

Comment:
Don't put a link like this in the documentation. You can find some examples in different places to explain to you how to refer a page between components:
* In [showroom theme](https://bonitasoft.github.io/bonita-documentation-theme/links.html)
* In [Contribution guide](https://github.com/bonitasoft/bonita-documentation-site/blob/master/docs/content/CONTRIBUTING.adoc#adding-reference-between-pages-of-two-components-or-between-pages-of-2-versions-of-the-same-component)
* On another page of documentation (search  `xref:` syntax)

If you have any difficulty with this, ask for help in the #doc channel on Slack.

Reply:
Got it, I committed the changes